### ISSUE TITLE
Elassandra dies by a null pointer when broadcast_address is not defined

### ISSUE THREAD

**Issue by andre-paris** [strapdata/elassandra#204]
**Elasticsearch version**: 5.5.0.19

The error occurs only with the elasticsearch plugin enabled:
```
2018-06-25 10:41:28,216 ERROR [main] ElassandraDaemon.java:588 main Exception
java.lang.NullPointerException: null
	at java.util.Objects.requireNonNull(Objects.java:203)
	at org.elasticsearch.common.network.NetworkAddress.format(NetworkAddress.java:93)
	at org.elasticsearch.common.network.NetworkAddress.format(NetworkAddress.java:69)
	at org.elassandra.discovery.CassandraDiscovery.buildNodeName(CassandraDiscovery.java:137)
	at org.apache.cassandra.service.ElassandraDaemon.nodeSettings(ElassandraDaemon.java:453)
	at org.apache.cassandra.service.ElassandraDaemon.getSettings(ElassandraDaemon.java:436)
	at org.apache.cassandra.service.ElassandraDaemon.activate(ElassandraDaemon.java:187)
	at org.apache.cassandra.service.ElassandraDaemon.main(ElassandraDaemon.java:551)
```
@vroyer [warned me about the missing broadcast address configuration](https://github.com/strapdata/elassandra/issues/199#issuecomment-400185149), but the cassandra behavior is to adopt the value defined in listen_address as the default broadcast_addres as stated in the cassandra.yml comments:
```properties
# Address to broadcast to other Cassandra nodes
# Leaving this blank will set it to the same value as listen_address
# broadcast_address: 1.2.3.4
```
PS.: I tested with prior versions of Elassandra and Cassandra as well and the problem doesn't happens.

**Comment by vroyer** [comment]
Yes, this is fixed in this commit https://github.com/strapdata/elassandra/commit/51f10b58d00d3158bb8e46890cfceb05c2cfa2df  . 

If you don't want to rebuild and redeploy elassandra before the next release, the workaround is to explicitly set your broadcast_address in your cassandra.yaml to avoid this NPE.
Thanks.

**Comment by vroyer** [comment]
Fixed in 5.5.0.19
